annotate.el

```diff
@@ -2076,15 +2082,26 @@ from a chain where `annotation' belong."
        (t
         (move-overlay last-annotation last-annotation-starting-pos new-ending-pos))))))
 
-(defun annotate--delete-annotation-prevent-modification (annotation)
+(defun annotate--delete-annotation-chain-prevent-modification (annotation)
+"Delete an annotation chain backing up and restoring modification
+status of the buffer before deltion occured."
```

Comment:
typo: "deletion"

Reply:
Hi @bastibe !

> Another great addition, thank you!

Thanks!

> I am not quite sure whether it is a good idea to prompt the user when deleting annotations. It is certainly something I would disable immediately. But if you find it useful, that's reason enough.

The funny thing is that i find asking any time annoying too! :-D, i am very happy to switch the default behaviour to not ask for confirmation! :-)

Bye!
C.